__tests__/app.test.js

```diff
@@ -65,3 +66,40 @@ describe("GET /api/reviews", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id", () => {
+  test("200 : responds with review object containing correct properties", () => {
+    return request(app)
+      .get("/api/reviews/2")
+      .expect(200)
+      .then((result) => {
+        expect(result.body.review).toMatchObject({
+          review_id: 2,
```

Comment:
I like how you checked specifically for this id as we know this must be 2, but you can also make the same assertions for the rest of the data to confirm everything is exactly what it should be instead of just generic strings and numbers in this test.